src/stores/game.js

```diff
@@ -172,6 +173,11 @@ export const useGameStore = defineStore('game', {
           this.cardChosenFromScrap = null;
           this.playerChoosingFromScrap = null;
         }
+        if (Object.hasOwnProperty.call(newGame.lastEvent, 'discardedCards')) {
+          this.discardedCards = newGame.lastEvent.discardedCards;
+        } else {
+          this.discardedCards = null;
+        }
```

Comment:
I'm not sure this is actually needed, I think the code would work just fine without it, especially since we explicitly set `this.discardedCards` in the `processFours()` function in the store. But I put this as a safety so that it will null out `this.discardedCards` if it wasn't the last event to happen

Reply:
Safety first, kids